setup-compose.sh

```diff
@@ -1,12 +1,23 @@
 #!/bin/bash
-if [ "$1" == '--eval' ]; then
-    export RELEASE="eval"
-else
-    export RELEASE="latest"
-fi
 
-## TODO: This should not build the images by default, but should just download them
-make build
+# Initialize variables
+export RELEASE="latest"
+export SOURCE="binpash/"
 
+for arg in "$@"; do
+    if [ "$arg" == '--rebuild' ]; then
```

Comment:
That's great, though there is a small problem with this code. If one passes --rebuild twice, it will `make build` twice. You can probably set a variable FLAG_REBUILD=1 if you find this argument and then if flag is one, `make build` outside of the loop!

Reply:
I fixed it.